pokeapp/src/components/PokemonCard.js

```diff
@@ -105,6 +127,7 @@ const PokemonCard = ({
         <h3>#{id}</h3>
         {pokemon.quantity > 0 ? <p>{pokemon.quantity}</p> : ""}
       </div>
+      <button onClick={() => console.log(myPokeList)}> mypokeList</button>
```

Comment:
Looks good. Just wondering if you still need this log here. Keep up the good work!

Reply:
I was using it to test the functionality. I will remove it. Thanks for catching it!